HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -129,11 +129,26 @@ def customiseFor2018Input(process):
 
     return process
 
+def customiseFor33495(process):
```

Comment:
Should be customiseFor33792.

However, could this customisation and changes in this file just be removed, and the parameters in question be labelled "obsolete" in fillDescriptions of the plugins, such that configs containing them would still work?
Reason is that online does not use this customisation, and we also do not want to break old 11_3 menus.

Reply:
@Martin-Grunewald, commit https://github.com/cms-sw/cmssw/pull/33792/commits/2044fd55de151707a1bfbf508e76345d16508ce4 addresses your comment and incidentally remove the HLT pending flag.